core/Templating/AdapterFileSystem.cs

```diff
@@ -0,0 +1,131 @@
+﻿using Microsoft.TemplateEngine.Abstractions.PhysicalFileSystem;
+using Microsoft.TemplateEngine.Utils;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.IO.Abstractions;
+
+namespace Cmf.CLI.Core.Templating
+{
+    /// <summary>
+    /// Adapter to allow using an <see cref="IFileSystem"/> anywhere where a <see cref="IPhysicalFileSystem"/> was expected.
+    /// </summary>
+    public class AdapterFileSystem : IPhysicalFileSystem
```

Comment:
nitpick: I'd name this to refer to the IO.Abstractions library, as it is only compatible with that.

Reply:
Renamed to `IOAbstractionsFileSystem`